src/vs/platform/windows/common/windows.ts

```diff
@@ -295,7 +295,7 @@ export enum ReadyState {
 export interface IPath {
 
 	// the file path to open within a Code instance
-	filePath?: string;
+	fileUri?: URI;
```

Comment:
@aeschli I would do:

```typescript
export interface IPath extends IPathData {

	// the file path to open within a Code instance
	fileUri?: URI;
}
```

Reply:
As you like, I personally find such hierarchies of interfaces difficult to read.